.github/workflows/docker-image.yml

```diff
@@ -1,22 +1,74 @@
 name: Docker Image CI for shopify-tracker
-
 on:
   push:
-    branches: [ master ]
+    branches: [ main ]
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
 
-jobs:
+permissions: read-all
 
+jobs:
```

Comment:
All the jobs must run only if the PR is merged.

Reply:
done